src/backend/graphItemService.ts

```diff
@@ -307,4 +317,30 @@ export class GraphItemService {
       badRows: [[RowErrorMessage.SOURCE_TARGET_NOT_FOUND, noExtremitiesRows]]
     };
   }
+
+  /**
+   * Ensure that the current user has the correct privileges to import nodes and create a report of the nodes imported.
+   *
+   * @param sourceKey
+   * @param rc
+   */
+  private static async assertHasReportNodeAccess(rc: RestClient, sourceKey: string): Promise<void> {
```

Comment:
maybe changing the name of the method to assertCanReadReportNode

Reply:
done